### ISSUE TITLE
Add options to work only on repositories or on the AUR

### ISSUE THREAD

**Issue by Exagone313** [Jguer/yay#311]
Hi,

I'm coming from pacaur, and I liked the options `-r` and `-a` that, respectively, restrict operations to repositories or the AUR. I often upgrade only from the repositories, and having to type `aur` to filter out AUR upgrades each time is annoying. I'd prefer to type the command `yay -Syur` instead. I know partial upgrades are not supported on Arch, but AUR packages are not supported anyway.

This also applies to other operations such as searching, and this is handy.

I haven't found similar options in yay, or issues/pull requests asking for this feature.

**Comment by Morganamilo** [comment]
I always thought doing `yay -S aur/packagename` as well as typing `aur` during upgrade, which you say you find annoying, was good enough so I didn't see a point in adding a flag. And if you're just upgrading repo packages you can just use Pacman directly.

Plus `-r --root` is a Pacman flag, so Tay will never overwrite that so you'd have to write `--repo` which is just as annoying as typing AUR in my opinion.

`-r --root` is schedule for removes for removal iirc so maybe then it could be added.

Basically my question is, you want this for convenience right? If you had to enter `yay -Syu --repo` would it still be as convenient and worth it?

**Comment by Exagone313** [comment]
if `-r` is already taken, and anotehr short option isn't wanted, a long option is fine too.

**Comment by passcod** [comment]
Mostly interested in the inverse for this: being able to tell yay _not_ to target the AUR for something. I know I could just use pacman directly, but then I don't have the yay improvements and otherwise brevity.

**Comment by passcod** [comment]
Specifically, something like `yay -Syun` (`-n` for "not AUR", doesn't conflict afaik).
